[2024-08-21T06:32:31.273+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-21T06:32:31.298+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_22024-08-21.fetch_the_api scheduled__2024-08-20T00:00:00+00:00 [queued]>
[2024-08-21T06:32:31.307+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_22024-08-21.fetch_the_api scheduled__2024-08-20T00:00:00+00:00 [queued]>
[2024-08-21T06:32:31.308+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-21T06:32:31.334+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): fetch_the_api> on 2024-08-20 00:00:00+00:00
[2024-08-21T06:32:31.339+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=4792) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-21T06:32:31.340+0000] {standard_task_runner.py:63} INFO - Started process 4796 to run task
[2024-08-21T06:32:31.340+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_22024-08-21', 'fetch_the_api', 'scheduled__2024-08-20T00:00:00+00:00', '--job-id', '785', '--raw', '--subdir', 'DAGS_FOLDER/final_etl.py', '--cfg-path', '/tmp/tmpbqt5p2kh']
[2024-08-21T06:32:31.342+0000] {standard_task_runner.py:91} INFO - Job 785: Subtask fetch_the_api
[2024-08-21T06:32:31.490+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_22024-08-21.fetch_the_api scheduled__2024-08-20T00:00:00+00:00 [running]> on host 7e1333882d71
[2024-08-21T06:32:31.623+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_22024-08-21' AIRFLOW_CTX_TASK_ID='fetch_the_api' AIRFLOW_CTX_EXECUTION_DATE='2024-08-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-20T00:00:00+00:00'
[2024-08-21T06:32:31.626+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-21T06:32:31.639+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-21T06:32:31.641+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/***/data/config.py']
[2024-08-21T06:32:31.649+0000] {subprocess.py:86} INFO - Output:
[2024-08-21T06:32:31.724+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-21T06:32:31.726+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-21T06:32:31.753+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_22024-08-21, task_id=fetch_the_api, run_id=scheduled__2024-08-20T00:00:00+00:00, execution_date=20240820T000000, start_date=20240821T063231, end_date=20240821T063231
[2024-08-21T06:32:31.843+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-08-21T06:32:31.873+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-21T06:32:31.879+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
